amplify.yml

```diff
@@ -17,6 +17,7 @@ frontend:
         - yarn
         - yarn disable-telemetry
         - yarn gql
+        - timeout 1m yarn onesky:download || echo "yarn onesky:download timed out after 1 minute"
```

Comment:
```suggestion
        - timeout 1m yarn onesky:download || echo "yarn onesky:download failed or timed out after 1 minute"
```

Reply:
`onesky/download.js` catches most errors, logs them, and exits successfully, so it would take a syntax error or something like that to get it to print this message due to a non-timeout failure. But I can update it.